### PR TITLE
finished messages.json translation

### PR DESCRIPTION
**Pull request by ajyan**
# Add translation: Czech #14

## Description of the Change

- Added translations to the values in the `messages.json` file from English to Czech

## Applicable Issues

Add translation: Czech #14

### CODE REVIEW

**Comment by devmount** [comment]
Thank You Andrew for taking the time to contribute. At this time, this PR is incomplete, you only translated two values. Here are the missing translations:

image

Please complete the missing translations for this PR to be merged.

File: public/_locales/cs/messages.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+	"extensionDescription": {
+		"message": "Krásně vizualizované statistiky pro vaše e-mailové účty Thunderbird"
+	},
+	"popup": {
+		"message": "",
+		"account": "Účet | Účty",
+		"messagesInFolder": "{0} messages in {1} folders"
```
Comment (devmount) [inline public/_locales/cs/messages.json:8]: Translation is missing in all values from this line on.

**Review by devmount — CHANGES_REQUESTED** [review]

File: public/_locales/cs/messages.json (hunk 1 above)
Comment (ajyan) [reply public/_locales/cs/messages.json]: @devmount Must have pushed an old commit, I've updated the translations in commit: c22d182 and the final version is in commit cd26717.

**Comment by ajyan** [comment]
@devmount  Must have pushed an old commit, I've updated the translations in commit: c22d182 and the final version is in commit cd26717

File: public/_locales/cs/messages.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		"mailsTotal": "Celkem e-mailů",
+		"withinYears": "do {0} let",
+		"mailsUnread": "Nepřečtené e-maily",
+		"niceWork": "Pěkná práce!",
+		"percentOfReceived": "{0}% přijatých",
+		"mailsReceived": "Přijaté e-maily",
+		"percentOfTotal": "{0}% z celkového počtu",
+		"mailsSent": "E-maily odeslány",
+		"mailsPerMonth": "E-maily za měsíc",
+		"mailsYear": "mails/year",
```
Comment (devmount) [inline public/_locales/cs/messages.json:21]: Line 21 is still missing a translation, isn't it?

**Review by devmount — CHANGES_REQUESTED** [review]
Thank you for these changes, this looks much better! There still is line 21 missing, or am I wrong?

File: public/_locales/cs/messages.json (hunk 2 above)
Comment (ajyan) [reply public/_locales/cs/messages.json]: You are correct! Thanks for catching this and for working with me.

Just to confirm, `<a href='{0}' target='_blank'>Github</a>` should be left untranslated, is that correct?

Comment (ajyan) [reply public/_locales/cs/messages.json]: Edits to mails per year in b644d36

**Review by devmount — APPROVED** [review]
Thanks again for improving this PR. Yes indeed `<a href='{0}' target='_blank'>Github</a>` should be left as you did.

**Pull request merged**